cnv/upgrading-container-native-virtualization.adoc

```diff
@@ -4,14 +4,17 @@ include::modules/cnv-document-attributes.adoc[]
 :context: upgrading-container-native-virtualization
 toc::[]
 
-You enable automatic updates during {CNVProductName} xref:../cnv/cnv_install/installing-container-native-virtualization.adoc#cnv-subscribing-to-the-catalog_installing-container-native-virtualization[installation].
-Learn what to expect and how you can check the status of an update in progress.
+* Learn how to:
```

Comment:
Are lines 7-9 necessary? I suspect we can go straight into the tech preview warning.
However, if yes, can we rewrite the opening bullet point into a complete sentence. Even if it's something like 
"You can manually upgrade to the next minor version of {CNVProductName), and monitor the status of an update, in the web console."